src/components/MetricsPane/MetricsPane.jsx

```diff
@@ -65,6 +70,8 @@ const MetricsPane = ({ showMetricsPane, setShowMetricsPane, showLoadingIndicator
   } = useFilterProjectsContext()
   const [selectedSampleEvent, setSelectedSampleEvent] = useState(null)
   const [metricsView, setMetricsView] = useState('summary')
+  const isMapView = view === 'mapView'
+  const map = useMap()[MAIN_MAP_ID] // the docs for react-map-gl are not clear on the return object for useMap. This is what works in testing. Further, its not a 'ref' its the actual map instance.
```

Comment:
Yup, its just all the docs lead me down the path of using `{current}`, so it wasted some time. Now that I know, its no biggie. There was also a 'default` property, but I didnt see mention of it in docs so opted to grab the map instance by id since its mentioned in docs.